api/client/commands.go

```diff
@@ -1656,13 +1657,35 @@ func (cli *DockerCli) CmdPs(args ...string) error {
 	}
 
 	w := tabwriter.NewWriter(cli.out, 20, 1, 3, ' ', 0)
+	if *quiet {
+		*fields = "c"
+	}
+
+	if *size {
+		*fields = *fields + "z"
```

Comment:
It was my consideration that it could be desired, but doesn't seem in line with what I think a user would suspect. I'll uniq the letters that are set in *fields, choosing the earliest existence.